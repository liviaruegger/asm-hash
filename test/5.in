Bom, ia falando! questao, isso que me sovaca... Ah, formei aquela pergunta, para compadre meu Quelemem. Que me respondeu! que, por perto do Ceu, a gente se alimpou tanto, que todos os feios passados se exalaram de nao ser - feito sem-modez de tempo de crianca, mas-artes. Como a gente  nao carece de ter remorso do que divulgou no latejo de seus pesadelos de uma noite.
