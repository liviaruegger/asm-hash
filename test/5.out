e7ebf435d398ed9c06fe27b9cecceb13
